doc/contributing/collaborator-guide.md

```diff
@@ -232,7 +232,6 @@ There are some other files that touch the build chain. Changes in the following
 files also qualify as affecting the `node` binary:
 
 * `tools/*.py`
-* `tools/build-addons.mjs`
 * `*.gyp`
```

Comment:
```suggestion
* `tools/build-addons.py`
* `*.gyp`
```

Reply:
There is already a `tools/*.py` item in the list, I think we don't need to list `tools/build-addons.py` separately?